docs/Tutorial.md

```diff
@@ -604,7 +592,7 @@ But now that the `users` resource has a `show` view, you can also link to it fro
 // in src/posts.tsx
 export const PostList = () => (
     <List>
-        <Datagrid rowClick="edit">
+        <Datagrid>
 -           <ReferenceField source="userId" reference="users" />
 +           <ReferenceField source="userId" reference="users" link="show" />
```

Comment:
Not for this PR but this should probably be automatic as well (detection of the available views with the same priorities)